### PR TITLE
Fixed broken links to plugins in main README.md file

### LINKED ISSUE #1304: Link for PagerDuty in the main README.md is broken
### What do you want to improve?

The link for the PagerDuty plugin is broken in https://github.com/janus-idp/backstage-showcase/blob/main/README.md#features and should be updated

### PR DESCRIPTION
**Pull request by 04kash**
## Description

Fixed broken links to plugins in main README.md file for the following:
- GitHub Pull Requests plugin
- GitHub Actions plugin
- GitHub Issues plugin
- Azure DevOps plugin
- Jenkins plugin
- PagerDuty
- Lighthouse plugin
- Dynatrace plugin

## Which issue(s) does this PR fix

- Fixes #1304 

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by gashcrumb — APPROVED** [review]
:+1:

File: README.md [hunk 1]
```diff
@@ -22,23 +22,23 @@ Our current list of plugins within the showcase app include:
...
+- [GitHub Issues plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/github-issues)
 - [GitHub Discovery](https://backstage.io/docs/integrations/github/discovery) & [Org Data](https://backstage.io/docs/integrations/github/org)
 - [Security Insights plugin](https://github.com/RoadieHQ/roadie-backstage-plugins/tree/main/plugins/frontend/backstage-plugin-security-insights)
 - [Keycloak plugin](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/keycloak-backend)
 - [SonarQube plugin](https://roadie.io/backstage/plugins/sonarqube/)
 - [Tekton plugin](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/tekton)
-- [Azure DevOps plugin](https://github.com/backstage/backstage/blob/master/plugins/azure-devops/README.md)
-- [Jenkins plugin](https://github.com/backstage/backstage/tree/master/plugins/jenkins)
+- [Azure DevOps plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/azure-devops)
+- [Jenkins plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/jenkins)
```
Comment (Zaperex) [inline README.md:34]:
```suggestion
- [Azure DevOps plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/azure-devops/plugins/azure-devops/README.md)
- [Jenkins plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/jenkins/plugins/jenkins/README.md)
```

File: README.md [hunk 2]
```diff
@@ -22,23 +22,23 @@ Our current list of plugins within the showcase app include:
 - [Topology plugin](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/topology)
 - [ArgoCD plugin](https://github.com/RoadieHQ/roadie-backstage-plugins/tree/main/plugins/frontend/backstage-plugin-argo-cd)
 - [GitHub Insights plugin](https://github.com/RoadieHQ/roadie-backstage-plugins/tree/main/plugins/frontend/backstage-plugin-github-insights)
-- [GitHub Pull Requests plugin](https://github.com/backstage/backstage/tree/master/plugins/github-pull-requests-board)
-- [GitHub Actions plugin](https://github.com/backstage/backstage/tree/master/plugins/github-actions)
-- [GitHub Issues plugin](https://github.com/backstage/backstage/tree/master/plugins/github-issues)
+- [GitHub Pull Requests plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/github-pull-requests-board)
+- [GitHub Actions plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/github-actions)
+- [GitHub Issues plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/github-issues)
```
Comment (Zaperex) [inline README.md:27]:
```suggestion
- [GitHub Pull Requests plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/github-pull-requests-board/plugins/github-pull-requests-board/README.md)
- [GitHub Actions plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/github-actions/plugins/github-actions/README.md)
- [GitHub Issues plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/github-issues/plugins/github-issues/README.md)
```

File: README.md [hunk 3]
```diff
@@ -22,23 +22,23 @@ Our current list of plugins within the showcase app include:
...
 - [GitLab plugin](https://github.com/immobiliare/backstage-plugin-gitlab)
 - [Jira plugin](https://github.com/RoadieHQ/roadie-backstage-plugins/tree/main/plugins/frontend/backstage-plugin-jira)
 - [Jfrog Artifactory plugin](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/jfrog-artifactory)
 - [Datadog plugin](https://github.com/RoadieHQ/roadie-backstage-plugins/tree/main/plugins/frontend/backstage-plugin-datadog)
-- [PagerDuty](https://github.com/backstage/backstage/tree/master/plugins/pagerduty)
-- [Lighthouse plugin](https://github.com/backstage/backstage/tree/master/plugins/lighthouse)
-- [Dynatrace plugin](https://github.com/backstage/backstage/tree/master/plugins/dynatrace)
+- [PagerDuty](https://github.com/PagerDuty/backstage-plugin)
+- [Lighthouse plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/lighthouse)
+- [Dynatrace plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/dynatrace)
```
Comment (Zaperex) [inline README.md:41]:
```suggestion
- [Lighthouse plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/lighthouse/plugins/lighthouse/README.md)
- [Dynatrace plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/dynatrace/plugins/dynatrace/README.md)
```

**Review by Zaperex — CHANGES_REQUESTED** [review]
Can we link to the plugin instead of to the workspace the plugins are located in?

**Review by Zaperex — APPROVED** [review]
/lgtm

**Review by Zaperex — APPROVED** [review]
/lgtm

**Pull request merged**